src/kuzya.cpp

```diff
@@ -183,14 +140,68 @@ Kuzya::Kuzya(QWidget *parent)
 
     RFileList = new QList<QString>();
 
+    myFile = nullptr;
```

Comment:
Move to initializer list of Kuzya constructor.

Reply:
It is in the constructor of Kuzya. Am i wrong?